test/types/file/unableToOpenFile.good

```diff
@@ -1 +1 @@
-unableToOpenFile.chpl:1: error: No such file or directory in open with path "_test_cannotOpenMe.txt"
+unableToOpenFile.chpl:1: error: uncaught error - No such file or directory in open with path "_test_cannotOpenMe.txt"
```

Comment:
While I like the second style you offer, the `error: ` comes from the `chpl_error` primitive called to halt on an uncaught error. This would require a new runtime function, which might be worthwhile.

After talking with @lydia-duncan about this, we preferred that the type of the error (e.g. `SysError`) be included in the message at some point. Here's one concept:

```
unableToOpenFile.chpl:1: uncaught SystemError - No such file or directory in open with path "_test_cannotOpenMe.txt"
```

In the ideal world, we might want to look at how Java does [stack traces on exceptions](https://en.wikibooks.org/wiki/Java_Programming/Stack_trace) as a starting point. If I understand what you said correctly, I agree that we should show where the error was initially created and where it was not caught. Something like:

```
uncaught SystemError in thread 0 on locale 0:
    at unableToOpenFile.main() [unableToOpenFile:1]
    at [where the error was created]
```

A stack trace would fill in the `throw` locations between those.